### PR TITLE
Analytics View Helper

### PR DESCRIPTION
**Pull request by waltertamboer**

### CODE REVIEW

File: config/module.config.php [hunk 1]
```diff
@@ -6,6 +6,10 @@
         ],
     ],
     'phpab' => [
+        'analytics' => [
+            'collector' => null,
```
Comment (waltertamboer) [inline config/module.config.php]: By default we don't configure a data collector and renderer. We leave this up to user-land. Maybe the user wants to use PDO analytics?

We could argue to set this up by default. I'm up for suggestions.

**Comment by pachico** [comment]
Once we have the DataCollector\Generic we could set it as default.
Renderer is very user specific, instead.

File: config/module.config.php (hunk 1 above)
Comment (pachico) [inline config/module.config.php]: Once we have the DataCollector\Generic we could set it as default.
Renderer is very user specific, instead.

Comment (waltertamboer) [inline config/module.config.php]: Good idea :+1:

**Pull request merged**